### DISCUSSION TITLE
Can we use new react compiler with page router ?

### DISCUSSION THREAD

**Discussion by lucas-garrido** [vercel/next.js#73598] [App Router] [answered]
Hello !
This is not mentioned in the documentation, can we expect this feature to be available in page router ?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

Yes it is supported, it is not a thing of only App Router. Odd that the docs for Pages Router did not pick up on this.

<img width="677" alt="Screenshot 2024-12-06 at 19 52 43" src="https://github.com/user-attachments/assets/914c307b-38da-4530-86bb-bcad69f2af26">

Here's a Stackblitz example using it, with Next 15 and React 19 mind you: https://stackblitz.com/edit/nextjs-zdkojg?file=pages%2Findex.tsx,next.config.js,package.json (notice the ✨ next to the Page component)
